channels/us.m3u

```diff
@@ -6526,3 +6526,59 @@ https://ln-zen.localnowlive.com/v1/master/385c85a93929f94966d0fb186fc33b431e6f1e
+#EXTINF:-1 tvg-id="WWL.us" tvg-name="CBS 4 New Orleans LA (WWL)" tvg-country="US" tvg-language="English" tvg-logo="" group-title="Local",CBS 4 New Orleans, LA (WWL)
+https://livevideo01.wwltv.com/hls/live/2016516/newscasts/live.m3u8
+#EXTINF:-1 tvg-id="WHAS.us" tvg-name="ABC 11 Louisville KY (WHAS)" tvg-country="US" tvg-language="English" tvg-logo="" group-title="Local",ABC 11 Louisville, KY (WHAS)
+https://livevideo01.whas11.com/hls/live/2016284/newscasts/live.m3u8
+#EXTINF:-1 tvg-id="WTHR.us" tvg-name="NBC 13 Indianapolis IN (WTHR)" tvg-country="US" tvg-language="English" tvg-logo="" group-title="Local",NBC 13 Indianapolis, IN (WTHR)
+https://livevideo01.wthr.com/hls/live/2013835/newscasts/live.m3u8
+#EXTINF:-1 tvg-id="KTVB.us" tvg-name="KTVB 7 Boise ID" tvg-country="US" tvg-language="English" tvg-logo="" group-title="Local",KTVB 7 Boise, ID
+https://livevideo01.ktvb.com/hls/live/2014542/newscasts/live.m3u8
+#EXTINF:-1 tvg-id="WOI.us" tvg-name="ABC 5 Des Moines IA (WOI)" tvg-country="US" tvg-language="English" tvg-logo="" group-title="Local",ABC 5 Des Moines, IA (WOI)
+https://livevideo01.weareiowa.com/hls/live/2011593/newscasts/live.m3u8
+#EXTINF:-1 tvg-id="WQAD.us" tvg-name="ABC 8 Davenport IA (WQAD)" tvg-country="US" tvg-language="English" tvg-logo="" group-title="Local",ABC 8 Davenport, IA (WQAD)
+https://livevideo01.wqad.com/hls/live/2011657/newscasts/live.m3u8
+#EXTINF:-1 tvg-id="WMAZ.us" tvg-name="CBS 13 Macon GA (WMAZ)" tvg-country="US" tvg-language="English" tvg-logo="" group-title="Local",CBS 13 Macon, GA (WMAZ)
+https://livevideo01.13wmaz.com/hls/live/2017376/newscasts/live.m3u8
+#EXTINF:-1 tvg-id="WXIA.us" tvg-name="NBC 11 Alive Atlanta GA (WXIA)" tvg-country="US" tvg-language="English" tvg-logo="" group-title="Local",NBC 11 Alive Atlanta, GA (WXIA)
+https://livevideo01.11alive.com/hls/live/2015499/newscasts/live.m3u8
+#EXTINF:-1 tvg-id="WTSP.us" tvg-name="CBS 10 Tampa FL (WTSP)" tvg-country="US" tvg-language="English" tvg-logo="" group-title="Local",CBS 10 Tampa, FL (WTSP)
+https://livevideo01.wtsp.com/hls/live/2015503/newscasts/live.m3u8
+#EXTINF:-1 tvg-id="WTLV.us" tvg-name="NBC Jacksonville FL (WTLV)" tvg-country="US" tvg-language="English" tvg-logo="" group-title="Local",NBC / ABC Jacksonville, FL (WTLV)
+https://livevideo01.firstcoastnews.com/hls/live/2014550/newscasts/live.m3u8
+#EXTINF:-1 tvg-id="WTIC.us" tvg-name="FOX 61 Hartford CT (WTIC)" tvg-country="US" tvg-language="English" tvg-logo="" group-title="Local",FOX 61 Hartford, CT (WTIC)
+https://livevideo01.fox61.com/hls/live/2011658/newscasts/live.m3u8
+#EXTINF:-1 tvg-id="KUSA.us" tvg-name="NBC 9 Denver CO (KUSA)" tvg-country="US" tvg-language="English" tvg-logo="" group-title="Local",NBC 9 Denver, CO (KUSA)
+https://livevideo01.9news.com/hls/live/2014548/newscasts/live.m3u8
+#EXTINF:-1 tvg-id="CWNetworkSanDiego.us" tvg-name="CW Network San Diego" tvg-country="US" tvg-language="English" tvg-logo="" group-title="Local",CW Network San Diego
```

Comment:
Is it really the CW Network ? I see CBS 8 San Diego (KFMB)

Reply:
I just rename it. It really channel 8.2 KFMB-TV2. The Grit was like that yesterday.